### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by curtisbarnard** [f1db/f1db#2] [Q&A] [answered]
I was just thinking about starting my own project for F1 all-time statistics, but figured I should check and see if anyone has a project already going. It looks like you have some great data already. I'm really only experienced with frontend (HTML, CSS and JS) right now, but would love to help out. Is there anything I can do to help get a website up and running so people can filter and access the data?

**Comment by marceloverdijk** [comment, marked as answer]
Hi Curtis,

Thanks for you interest ;-)
We are currently working on a (static) website as next step in the F1DB portfolio. Probably it will be available in a month or two...
After that also a GraphQL api is on the TODO list.

Cheers,
Marcel

**Reply by alenvuletic** [reply]
@marceloverdijk Do we have a timeline for the API maybe?

**Reply by marceloverdijk** [reply]
My best estimate currently "before end of the year". But I hope to be able provide a working in progress openapi spec for who is interested soon.

**Reply by alenvuletic** [reply]
Awesome, thanks for the reply!
